src/client.rs

```diff
@@ -194,7 +190,16 @@ impl LastFm {
         params.insert("method".to_string(), operation.to_string());
         params.insert("api_sig".to_string(), signature);
 
-        self.http_client.post(url).form(&params).send()
+        let params: Vec<(&str, &str)> = params
```

Comment:
I though the `HashMap<String, String>` is more convenient to manipulate headers while crafting a request, so we only do this conversion at the last minute.